ndc-lambda-sdk/src/host.ts

```diff
@@ -1,3 +1,7 @@
+// We must initialize OpenTelemetry instrumentation before importing any other module
+import * as sdkInstrumentation from "@hasura/ndc-sdk-typescript/instrumentation";
+sdkInstrumentation.initTelemetry("ndc-lambda-sdk");
```

Comment:
Gross but we do what must be done.

Reply:
<img src="https://github.com/hasura/ndc-nodejs-lambda/assets/1214352/6495a01e-06ab-426e-b972-a336867afcf9" width="400" />